PreventecUCR/monitorsngr
# PREV-5-PGE: Títulos creados automáticamente para los compromisos PGE.

Steps to test:
- `drush master-scope local; drush master-execute -y; drush fra -y; drush cc all`
-  As institutional user go to one existing  emergency decree (`/decreto/decreto_emergencia_proteger_dengue_chikungunya_zika`)
-  use the button **"Add PGE Compromise"**  to add a new compromise to the Emergency decree
- Now in the creation of the PGE Compromise you should see the fields Emergency Decree and title filled with values related with the Emergency Decree.
